index.html

```diff
@@ -7,7 +7,7 @@
     </head>
     <body>
         <header>
-            Company Logo
+            Company Logo Here
```

Comment:
Can you please wrap this in a div?
I also added a suggestion to modify the content
You can apply my suggestion by clicking on "commit suggestion"
After that run locally git pull origin feature/search-2
Then make changes I requested (add a wrapping div)
Then add commit push
Come back here to see the changes
Request a review from me again by clicking on ♻️ icon next to my name in reviewers
```suggestion
            Company Logo Right Here
```